test/externalTests.sh

```diff
@@ -79,6 +79,7 @@ function test_truffle
       fi
       # Change "compileStandard" to "compile"
       sed -i s/solc.compileStandard/solc.compile/ "node_modules/truffle/build/cli.bundled.js"
+      npx truffle compile
```

Comment:
What is npx?

Reply:
`npm execute` - learned about it from the travis file inside gnosis/safe-contracts